src/tests/suites/Logger-Test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from "chai";
+import { beforeEach, describe, it } from "mocha";
+import { FileLogWriter } from "../../logger/FileLogWriter";
+import { Logger } from "../../logger/Logger";
+import Sinon = require("sinon");
```

Comment:
it's unfortunate that sinon doesn't support es6 imports

Reply:
I looked into it, there are issues open for it but didn't find them working yet.